runtime/polkadot/src/lib.rs

```diff
@@ -1130,48 +1277,63 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPallets,
-	TechnicalMembershipStoragePrefixMigration,
```

Comment:
Did I remove this prematurely or was it good to remove?

Reply:
On the current Polkadot network (9090), this migration has NOT happened yet.

Will these changes go into v0.9.10? If so, then yeah, we should have both in there.

Reply:
I added this back for now